pkgs/development/tools/build-managers/gn/apple_apsl.patch

```diff
@@ -0,0 +1,786 @@
+diff -uNr a.orig/third_party/apple_apsl/CFBase.h a/third_party/apple_apsl/CFBase.h
```

Comment:
Is it possible to use `fetchpatch` here?

Reply:
well, generally yes.

After all the patch is actually sources from Google's third_party repository. The problem with the repository is that it's currently ~2,5GB in size.

So I see three, maybe four options:

# just clone the repository and ignore the traffic
# use patch included with nixpkgs (like currently done)
# host patch somewhere else and use `fetchpatch` (question is where to host)
# create new github repo and put that stuff there

Problem with the latter two is that I'd have to put that stuff somewhere and link to there. I don't expect many changes to those, yet I'd have to update the resources ... or someone else host them at yet another place

I'm uncertain what the best option is ...

Reply:
Do they support a svn checkout of just the subdirectory?

Reply:
I found that: https://chromium.googlesource.com/chromium/src/third_party/+/7b93eb6514e15b92eb8d8b1767b92612c12c3364/apple_apsl/ they provide a tarball download:
https://chromium.googlesource.com/chromium/src/third_party/+archive/7b93eb6514e15b92eb8d8b1767b92612c12c3364/apple_apsl.tar.gz

Reply:
neat ... I've changed it to `fetchurl` from that url.  Thanks :)